assembler/assembler.c

```diff
@@ -642,6 +655,20 @@ object_t *assemble(FILE *file, const char *file_name, assembler_settings_t *sett
 				stack_pop(state.source_map_stack);
 				scas_log(L_INFO, "Returning to file '%s'", (char *)stack_peek(state.file_name_stack));
 			} else {
+				for (int i = 0; i<state.object->unresolved->length; i++) {
+					unresolved_symbol_t *sym = (unresolved_symbol_t*)state.object->unresolved->items[i];
+					bool imported = false;
+					for (int j = 0; j<state.object->imports->length; j++) {
```

Comment:
< operator needs spaces around it, too. Also >, <=, >=, and !=, and all arithmetic operators.